src/org/labkey/test/util/DeferredErrorCollector.java

```diff
@@ -37,14 +42,32 @@ public DeferredErrorCollector(@NotNull BaseWebDriverTest test)
         this(test.getArtifactCollector());
     }
 
+    /**
+     * Reset this error checker to
+     */
+    public void resetErrorTypes()
+    {
+        errorTypes.clear();
+        errorTypes.add(AssertionError.class);
+    }
+
+    public void addRecordableErrorType(Class<? extends Throwable> errorType)
```

Comment:
JavaDocs comment for public interfaces?

Reply:
Yup. Got half way through the javadoc for these new methods and forgot to get back to them.